server/tso/global_allocator.go

```diff
@@ -129,6 +127,14 @@ func (gta *GlobalTSOAllocator) GenerateTSO(count uint32) (pdpb.Timestamp, error)
 		currentGlobalTSO pdpb.Timestamp
 		err              error
 	)
+	// Differentiate the Global TSO synced from all the other Local TSOs
+	serialNum := gta.allocatorManager.getSortedDCLocationLength()
+	// log2(the number of dc-locations + global)
+	shiftNum := int(math.Ceil(math.Log2(float64(serialNum + 1))))
```

Comment:
I tend to use a fixed `shiftNum` instead of a dynamic one. What about set it as `8`? I think `2^8` can hold most of the situations.